nservicebus/upgrades/5to6.md

```diff
@@ -45,15 +45,15 @@ WARNING: Do not `return null` from the message handlers. A `null` will result in
 
 snippet:5to6-messagehandler
 
+For a step by step upgrade of existing handlers or sagas, see: [Migrate existing handlers/sagas](/nservicebus/upgrades/5to6-migrate-existing-handlers.md).
```

Comment:
Rather than remove the heading, why not put the "Migrate existing handlers" right below the handlers, i.e. right above, "Bus Send and Receive" ?